microsoftgraph/microsoft-graph-explorer-v4
# Task: Add device characteristics telemetry

## Overview

Closes #2001

## Demo
![image](https://user-images.githubusercontent.com/34104277/187243643-a57c9abd-5106-469f-b2a0-ada120e821f9.png)

## Testing Instructions

### How to test this PR
* Checkout to this branch.
* Load your local GE instance with the network section of the DevTools open.
* Run any query on GE.
* On the network section, filter the track requests and under Request Payload, find a data object with the baseType of "EventData".
* Under it, there is baseData with the name "DEVICE_CHARACTERISTICS_EVENT" with the device characteristics telemetry
* Resize the browser or change the device scale ratio and/or the browser zoom levels and observe telemetry.
* Run a query on GE and observe telemetry on the network section.
